.editorconfig

```diff
@@ -2,7 +2,7 @@
 root = true
 
 [*]
-indent_style = space
+indent_style = tab
```

Comment:
It is very confusing. Initially I used space based on this config file, but OAuthToken file used tab instead of space and that caused the wrong indentation. You can see the OAuthToken and it is using tab. I will switch this to space again anyway.